### PR TITLE
Wagtail inventory

### PR DESCRIPTION
**Pull request by Pomax**
Reinstated PR for https://github.com/mozilla/foundation.mozilla.org/pull/1640, which had to be backed out over a runtime flag incompatibility on staging.

/cc @alanmoo

### CODE REVIEW

File: Procfile [hunk 1]
```diff
@@ -1,2 +1,2 @@
-release: cd network-api && python ./manage.py migrate --no-input
+release: cd network-api && python ./manage.py migrate --no-input && python ./manage.py block_inventory --no-input
```
Comment (Pomax) [inline Procfile]: This `--no-input` flag breaks the staging deploy. I've filed cfpb/wagtail-inventory#15 to see if the maintainer of the library is interested in adding support for this flag, and hopefully that results in a new version that lets us add this plugin as part of the automated deployment.

Comment (alanmoo) [reply Procfile]: So this can't be reviewed until then?

**Comment by alanmoo** [comment]
Wait, I thought we were waiting for a change upstream. Why would this work if #1640 didn't?

**Comment by patjouk** [comment]
From what I understood, we don't need that "no-input" flag anymore. Can you confirm @Pomax ?

**Comment by Pomax** [comment]
indeed, after conferring with the library author, we should be able to run without `--no-input`

**Pull request merged**